lib/xlsx_reader/parsers/worksheet_parser.ex

```diff
@@ -312,13 +335,24 @@ defmodule XlsxReader.Parsers.WorksheetParser do
         {:ok, date_time} = Conversion.to_date_time(value, state.workbook.base_date)
         date_time
 
+      # Formulas
+
+      {"f", _, _} ->
+        %{:formula => state.formula, :value => state.value}
```

Comment:
To align the style with the rest of the code base, please use the shorthand notation for maps with atom keys, i.e. `%{formula: state.formula, value: state.value}`

Or, even better, apply `mix format` on your change.